modin/pandas/series.py

```diff
@@ -612,7 +611,9 @@ def error_raiser(msg, exception):
 
     agg = aggregate
 
-    def _apply(self, func, convert_dtype, args, **kwargs):  # noqa: PR01, RT01, D200
+    def apply(
+        self, func, convert_dtype=True, args=(), **kwargs
+    ):  # noqa: PR01, RT01, D200
```

Comment:
I don't think we need the `noqa`, docstring should be inherited

Reply:
I get a doc_checker error if I remove this noqa.